composer.json

```diff
@@ -60,7 +60,10 @@
         "symfony-web-dir": "web",
         "symfony-assets-install": "relative",
         "incenteev-parameters": {
-            "file": "app/config/parameters.yml"
+            "file": "app/config/parameters.yml",
+            "env-map": {
+                "database_url": "DATABASE_URL"
```

Comment:
you should make the ``secret`` configurable too (a random one should be generated when creating the new application if possible, and should not change for new deployments of the existing app)